server/v2/tests/users.test.js

```diff
@@ -0,0 +1,181 @@
+          done();
+        })
+    });
+    it('should return an object with an error when the user signs in without email or password', (done) => {
+      chai
+        .request(app)
+        .post('/api/v1/auth/signin')
+        .set('Content-type', 'application/json')
+        .set('Content-type', 'application/x-www-form-urlencoded')
+        .send({})
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).to.have.status(400)
+          expect(res.body).to.be.an('object')
+          expect(res.body).to.have.property('error')
+          done();
+        })
+    });
+    it('should return an object with a data -> token when the user signs in with valid credentials', (done) => {
+      const { email } = db.users[0];
+      chai
+        .request(app)
+        .post('/api/v1/auth/signin')
+        .set('Content-type', 'application/json')
+        .set('Content-type', 'application/x-www-form-urlencoded')
+        .send({email, password : 'usertest'})
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.an('object')
+          expect(res.body.data).to.have.property('token')
+          done();
+        })
+    });
+    it('should return an error with 400 status when signing in with an invalid password', (done) => {
+      const { email } = generator.users[1];
+      chai
+        .request(app)
+        .post('/api/v1/auth/signin')
+        .set('Content-type', 'application/json')
+        .set('Content-type', 'application/x-www-form-urlencoded')
+        .send({email, password : 'winnerstestwrong'})
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).to.have.status(400)
+          expect(res.body).to.be.an('object')
+          expect(res.body).to.have.property('error')
+          done();
+        })
+    });
+    it('should return an html page with 200 status when users access api documentation', (done) => {
+      const { email } = generator.users[1];
+      chai
+        .request(app)
+        .get('/api/v1/api-docs')
+        .set('Content-type', 'text/html')
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).to.have.status(200)
+          expect(res.headers['content-type']).to.equal('text/html; charset=utf-8')
+          done();
+        })
+    });
+    //The requested resource was not found on the server
+    it('should return an error with 404 status when the user accesses a wrong endpoint', (done) => {
+      const { email } = generator.users[1];
+      chai
+        .request(app)
+        .get('/api/v1111/wrong')
+        .set('Content-type', 'application/json')
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).to.have.status(404);
+          expect(res.body.error).to.equal(`The requested resource was not found on the server, Kindly read documentation here : ${docsUrl}`);
+          done();
+        })
+    });
+})
```

Comment:
Empty line required

Reply:
@oesukam is it now fixed, can it be merged?